orc8r/cloud/easy_deploy/docker/README.md

```diff
@@ -0,0 +1,198 @@
+*Steps for building Orc8r easy deploy image*
+
+1. Build the easy deployer image
+`docker build --pull  -t ksubraveti/orc8r_deployer:latest .`
```

Comment:
Goal is to push this into docker hub so that we don't have to build it to run the install process, that's the reason for keeping this separate. But I can do that check or pull within run.sh script